compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/service/errors/ErrorMessage.java

```diff
@@ -26,44 +26,50 @@
+                      + "accessor"),
+    ERROR_114("The GraphQL field `{0}` use input type `{1}` as an output type. A GraphQL field cannot use an input "
+                      + "type as an output type"),
+    ERROR_115("The GraphQL field `{0}` use output type `{1}` as an input type. A GraphQL field cannot use an output "
+                      + "type as an input type"),
+    ERROR_116("The graphql:Context should be the first parameter of method `{0}`"),
+    ERROR_117("Found path parameters `{0}` in GraphQL resource. Path parameters are not allowed in GraphQL resources"),
+    ERROR_118("Invalid resource path `{0}` found in GraphQL resource"),
+    ERROR_119("The graphql:Upload cannot be used as an input type of resource function `{0}`"),
+    ERROR_120("Multidimensional graphql:Upload array parameter found in function `{0}`. GraphQL input cannot have "
+                      + "multidimensional graphql:Upload arrays"),
     ERROR_121("Graphql input type must not be a subtype of `error?`"),
     ERROR_122("Invalid union type for GraphQL input type"),
-    ERROR_123("Invalid intersection type for GraphQL type"),
-    ERROR_124("All the resource functions in the GraphQL interface class `{0}` must be implemented in the child class" +
-                      " `{1}`"),
-    ERROR_125("Non-distinct service class `{0}` is used as a GraphQL interface"),
-    ERROR_126("Non-distinct service class `{0}` is used as a GraphQL interface implementation"),
-    ERROR_127("Invalid hierarchical resource path in subscribe resource"),
-    ERROR_128("GraphQL subscribe resource must return `stream` type"),
-    ERROR_129(
-            "Invalid GraphQL resource accessor. Only " + RESOURCE_FUNCTION_GET + " and " + RESOURCE_FUNCTION_SUBSCRIBE +
-                    " are allowed"),
-    ERROR_130("Failed to generate the schema from the service"),
-    ERROR_131("GraphQL interceptors can not have resource functions"),
-    ERROR_132("Invalid remote method in interceptor service. Only \"execute\" remote method is allowed"),
-    ERROR_133("Anonymous record `{0}` cannot be used as the type of the field `{1}`"),
-    ERROR_134("Anonymous record `{0}` cannot be used as an input object type of the field `{1}`");
+    ERROR_123("Non-distinct service class `{0}` is used as a GraphQL interface implementation"),
+    ERROR_124("Invalid hierarchical resource path `{0}` found in subscribe resource"),
+    ERROR_125("Invalid return type `{0}` found in subscribe resource `{1}`. GraphQL subscribe resource must return "
+                      + "`stream` type"),
+    ERROR_126("Invalid GraphQL resource accessor `{0}` found in resource `{1}`. Only " + RESOURCE_FUNCTION_GET +
+                      " and " + RESOURCE_FUNCTION_SUBSCRIBE + " are allowed"),
+    ERROR_127("Failed to generate the schema from the service. {0}"),
+    ERROR_128("Invalid resource method `{0}` found in GraphQL interceptor. GraphQL interceptors can not have resource "
+                      + "functions"),
```

Comment:
Let's make the wording consistent. The terms `resource method` and `resource function` both are used in the same message. Let's stick to one term and use it every place. I'd suggest we use the `resource method` and `remote method`, to keep consistency with the other documentation.